scripts/traffle.lic

```diff
@@ -82,326 +84,365 @@ module Traffle
       .map {|f| f.first.to_s.rjust(2, "0") + f.last }
       .reduce("") { |acc, col| acc + " " + col }
       .strip
-	end
+  end
 end
-
+load_towncrier = proc {
+  begin
+    mod = TownCrier_API.inspect
+  rescue NameError
+    Script.run("towncrier_api")
+  end
+}
 look_on_exclusions = ["disk","sign","board","list","kitten","cat","spirit","falcon","placard","rat","wolf","raven","figure","hamster"]
 
 timelimit = 15 #time in minutes to remove old raffles from list
 silence_me if !UserVars.traffle[:debug_my_script]
 Settings.load
 raffle_data = Settings['raffle_data'].sort_by {|date, ticket, room, time, lichid, price, quantity| time}
 if raffle_data == nil
-	raffle_data = Array.new
+  raffle_data = Array.new
 end
 
 timeleft = raffletime = raffleitem = raffle = output = n = roomname = absolute_time = raffle_cost = raffle_tickets = table_description = nil
 
 # pare old raffles
-raffle_data.reject! {|raffle| (raffle[3] - Time.now) < (-timelimit*60) }
+#raffle_data.reject! {|raffle| (raffle[3] - Time.now) < (-timelimit*60) }

```

Comment:
@mrhoribu Did you intend to comment this out? 

@rsecor Is this a change only meant for Town Crier updating?